index.html

```diff
@@ -48,6 +48,156 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+                <img src="./img/work_image.PNG" alt="">
+            </div>
+            <div class="card-content">
+                <h1>TONIC</h1>
+                <div><span><b>CANOPY</b></span> <span><img src="./img/circle.svg" class="circle" alt="dot">Back End
```

Comment:
- [ ] Please change the color of list-style (black circle ) in all cards to `#C1C7D0` as provided in figma and align them to the center. 
- [ ] Please decrease the font size to `13px`

Your design  | Figma design
------------- | -------------
<img width="405" alt="Screenshot 2022-03-18 at 14 27 34" src="https://user-images.githubusercontent.com/77038610/158986557-5b5d43a9-136e-40fc-8d61-97f59f9ddd2d.png"> | <img width="304" alt="Screenshot 2022-03-18 at 14 27 50" src="https://user-images.githubusercontent.com/77038610/158986599-be0f83ab-7813-47d8-86ac-d24eade27f5a.png">